### PR TITLE
Move the Gametile menu button alongside the Play button

### PR DESCRIPTION
**Pull request by lmeunier**
Not sure of this pull-request as this is an opinionated change.

The menu button, located at the top right corner of the thumbnail, is barely visible for some games. I think it would be better the have this button alongside of the action button (play/download/...).
Should we keep the previous icon ("applications-system-symbolic") or use the default icon for GtkMenuButton?

Also, the first letter of the action button is now capitalized.

Any comments are welcome (I'm not very confident with Gtk).

Before:
Screenshot from 2023-02-19 16-12-29

After:
Screenshot from 2023-02-19 16-11-17

### CODE REVIEW

**Comment by sharkwouter** [comment]
Oh nice, that's really cool!

File: minigalaxy/ui/gametile.py [hunk 1]
```diff
@@ -609,7 +614,7 @@ def __state_uninstalling(self):
     def __state_updatable(self):
         self.update_icon.show()
         self.update_icon.set_from_icon_name("emblem-synchronizing", Gtk.IconSize.LARGE_TOOLBAR)
-        self.button.set_label(_("play"))
+        self.button.set_label(_("Play"))
```
Comment (sharkwouter) [inline minigalaxy/ui/gametile.py:617]: This change makes it necessary to update all translation files. So we'd either have to leave this change out or update those.

File: minigalaxy/ui/gametile.py [hunk 2]
```diff
@@ -553,7 +556,7 @@ def __state_queued(self):
         self.__create_progress_bar()
 
     def __state_downloading(self):
-        self.button.set_label(_("downloading…"))
+        self.button.set_label(_("Downloading…"))
```
Comment (sharkwouter) [inline minigalaxy/ui/gametile.py:559]: This change makes it necessary to update all translation files. So we'd either have to leave this change out or update those.

File: minigalaxy/ui/gametile.py [hunk 3]
```diff
@@ -544,7 +547,7 @@ def __state_installable(self):
             self.progress_bar.destroy()
 
     def __state_queued(self):
-        self.button.set_label(_("in queue…"))
+        self.button.set_label(_("In queue…"))
```
Comment (sharkwouter) [inline minigalaxy/ui/gametile.py:550]: This change makes it necessary to update all translation files. So we'd either have to leave this change out or update those.

File: minigalaxy/ui/gametile.py [hunk 4]
```diff
@@ -618,7 +623,7 @@ def __state_updatable(self):
             self.wine_icon.set_margin_left(22)
 
     def __state_updating(self):
-        self.button.set_label(_("updating…"))
+        self.button.set_label(_("Updating…"))
```
Comment (sharkwouter) [inline minigalaxy/ui/gametile.py:626]: This change makes it necessary to update all translation files. So we'd either have to leave this change out or update those.

**Review by sharkwouter — CHANGES_REQUESTED** [review]
I really like this, it looks much better. Thanks for doing this!

I had one comment on the changes, which applies in a couple of places, so I made sure to make all of them. Besides that, could you update CHANGELOG.md as well? Then I'll merge it.

**Comment by lmeunier** [comment]
In order to update all translation files, all I have to do is run the script `./scripts/update-translation-files.sh`?

Should I also change the `gametilelist.ui/.py` files? To be honest, I was not aware that the game tiles could be displayed as a list, I just discovered this feature :)

**Comment by sharkwouter** [comment]
I'd say the best way to update the translations would actually be to go into the `.po` files and replace the translations for these words which are already there with the capitalized ones. Here would be an example: https://github.com/sharkwouter/minigalaxy/blob/766cc7c7071fdd4759c38991114a9ffe5da7646d/data/po/cs_CZ.po#L626

Just running the update script will not yield the result we want, as it will make new entries and the old ones will be ignored.

**Comment by lmeunier** [comment]
Translations for `msgid "Play"` have been added and some translations for `msgid "Install"` have been fixed.

**Comment by sharkwouter** [comment]
I've cherry-picked the commits which did not involve the updates to the strings and translation files. Thanks for this change! I don't think I can merge the rest of this right now. Maybe it would be better to have a separate PR for that part.

**Comment by lmeunier** [comment]
> Maybe it would be better to have a separate PR for that part.

I agree with you. I'll prepare a new PR with only the Capitalization of the first letter.
I think we can close this PR now.

**Comment by sharkwouter** [comment]
Cool, thanks again for the contribution!

**Pull request closed without merging**